### PR TITLE
Add new route for testing and package update for testing

### PR DESCRIPTION
**Pull request by wAndrewx**
## New Testing route 
- clears local db
- creates new account
- verifies it locally
 ## CORS roll back

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ app.use(
       tableName: "session",
     }),
     secret: process.env.SESSION_SECRET,
-    resave: false,
+    resave: true, //check if you want this
```
Comment (wAndrewx) [inline server.js]: i dont know if you want this to be true, it was false before

Comment (khoaHyh) [reply server.js]: I did want it to be false. Why did you want it to be true?

**Review by khoaHyh — CHANGES_REQUESTED** [review]
Do all existing routes work as intended with these new changes? In the future I'll just trust that you checked this.

**Review by khoaHyh — CHANGES_REQUESTED** [review]
revert resave

File: server.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const { pool } = require("./db/db");
 auth(passport);
 
 // Enable cors
-app.use(cors());
+app.use(cors({ credentials: true, origin: true }));
```
Comment (khoaHyh) [inline server.js]: make the origin our domain

**Review by khoaHyh — CHANGES_REQUESTED** [review]
see comment

**Pull request merged**